ext/js/background/backend.js

```diff
@@ -482,6 +482,7 @@ class Backend {
                 const noteIds = noteIdsArray[i];
                 if (noteIds.length > 0) {
                     cannotAdd[i].info.noteIds = noteIds;
+                    cannotAdd[i].info.noteInfos = await this._anki.notesInfo(noteIds);
```

Comment:
Since this information is not used in the default use case, let's add another argument like `getNoteInfos` to the signature of the `getAnkiNoteInfo`/`_onApiGetAnkiNoteInfo` API functions. Can probably start the passing it from here:

https://github.com/FooSoft/yomichan/blob/937e118da23da3e87c0c8433f5a2a94dee326864/ext/js/display/display.js#L1085-L1086